From 689223601ea8bd18b739bb722173cd00918b106d Mon Sep 17 00:00:00 2001
From: Yao Zi <ziyao@disroot.org>
Date: Sat, 26 Oct 2024 14:47:27 +0000
Subject: [PATCH] Add missing feature test macro to fix musl build

_POSIX_C_SOURCE should be defined to ensure declaration of kill(2)
is included. This fixes build on musl libc.

Link: https://www.man7.org/linux/man-pages/man2/kill.2.html#SYNOPSIS
Signed-off-by: Yao Zi <ziyao@disroot.org>
---
 src/gtklock.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/gtklock.c b/src/gtklock.c
index 2273506..f992c15 100644
--- a/src/gtklock.c
+++ b/src/gtklock.c
@@ -3,6 +3,7 @@
 
 // gtklock application
 
+#define _POSIX_C_SOURCE
 #include <signal.h>
 #include <gtk/gtk.h>
 
-- 
2.49.0

